File: 1.0/WearOS/WatchSprings-1.0/app/src/main/java/copyright/markmorris/allrightsreserved/watchsprings/functions/FunctionModel.kt
// WatchSprings 1.0 © 2025 Mark S. Morris. All rights reserved.
//
// https://www.watchsprings.com
//
// WatchSprings is Fan Content pursuant to the Epic Games Fan Content Policy. Portions of the materials used are trademarks and/or copyrighted works of Epic Games, Inc.

package copyright.markmorris.allrightsreserved.watchsprings.functions

import org.json.JSONArray
import org.json.JSONObject

class FunctionModel
{
    var isModelLoaded: Boolean = false

    lateinit var ItemEntries: JSONArray
    lateinit var ItemEntriesList: MutableList<Pair<String, JSONArray>>
    lateinit var ItemDetailID: String
    lateinit var ItemRowID: String
    lateinit var ItemType: String
    lateinit var ItemJSON: JSONObject

}

File: 1.0/watchOS/WatchSprings-1.0/WatchSprings WatchKit Extension/Functions/FunctionModel.swift
// WatchSprings 1.0 © 2025 Mark S. Morris. All rights reserved.
//
// https://www.watchsprings.com
//
// WatchSprings is Fan Content pursuant to the Epic Games Fan Content Policy. Portions of the materials used are trademarks and/or copyrighted works of Epic Games, Inc.

import WatchKit

class FunctionModel: NSObject
{
    var isModelLoaded: Bool = false
    
    var ItemEntries: NSMutableArray = []
    var ItemDetailID: String = "NoDetail"
    var ItemRowID: String = ""
    var ItemType: String = ""
    var ItemJSON: [String: Any] = [:]

}
